vmslave-config/pipeline/vmslave_config.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+git diff --dirstat=files ${GIT_PREVIOUS_COMMIT} | grep "vmslave-config/"
+if [ $? -eq 0 ]; then
+    mv vmslave-config/pipeline/all vmslave-config/ansible/group_vars/all
+    cd vmslave-config/ansible
+    ansible-playbook pr_gate_vmslave_config.yml
```

Comment:
which is best practice ?   
passing variable to ansible implicitly ?  or explicitly ?